qemu-test-init

```diff
@@ -11,19 +11,12 @@ mount -t tracefs tracefs /sys/kernel/tracing/ || true
 mount -t tmpfs none /mnt
 mount -t tmpfs none /tmp
 
-# FIXME
-MAKE_TARGET="check"
-
 BUILD_DIR="build/"
 
 # parse cmdline
 for x in $(cat /proc/cmdline); do
   if [ "$x" = "shell" ]; then
     SHELL=1
-  elif [ "$x" = "coverage" ]; then
-    MAKE_TARGET="check-code-coverage"
-  elif [ "$x" = "distcheck" ]; then
-    MAKE_TARGET="distcheck"
```

Comment:
We shouldn't remove these util we drop autotools support.

Reply:
We do not call `make` anymore in the script, thus these are useless anyway